Configuration/AlCa/python/autoCond.py

```diff
@@ -20,9 +20,11 @@
     # GlobalTag for MC production (Heavy Ions collisions) with optimistic alignment and calibrations for Run2
     'run2_mc_hi'        :   '80X_mcRun2_HeavyIon_v3',
     # GlobalTag for Run1 data reprocessing
-    'run1_data'         :   '80X_dataRun2_v4',
+    'run1_data'         :   '80X_dataRun2_v5',
```

Comment:
@slava77 
you're correct, v5 is already in the head of 80x.
Furthermore, take note of PR 13394, which we'll discuss at the ORP today, to move to v6.